app/static/src/app/components/DownloadOutput.vue

```diff
@@ -2,28 +2,28 @@
   <div>
     <div v-if="open" class="modal-backdrop fade show"></div>
     <div class="modal" :class="{show: open}" :style="style">
-      <div class="modal-dialog modal-dialog-centered">
+      <div class="modal-dialog modal-lg modal-dialog-centered">
         <div class="modal-content px-2">
           <div class="modal-header">
-            <h5 class="modal-title">Download Run</h5>
+            <h5 class="modal-title">Download {{ downloadType }}</h5>
```

Comment:
this is coming through as "Download Summary" in the sensitivity, but I think that it should be "Download summary" to mirror the casing of "Run sensitivity".

The icon also looks slightly too large to my eyes - this may vary by machine 
![image](https://github.com/mrc-ide/wodin/assets/1558093/27772b71-d631-4563-9c35-7655d2a4e701)

Reply:
Should be the same as the other Download button..?